src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -86,6 +86,7 @@
     @Nullable
     static volatile Function<? super Single, ? extends Single> onSingleAssembly;
 
+    @Nullable
```

Comment:
Do we also need the `@SuppressWarnings("rawtypes")` here?

Reply:
Completable does not have a type argument.